apps/web/src/services/cost-explorer/components/BudgetDetailBaseInformation.vue

```diff
@@ -78,17 +92,35 @@ const state = reactive({
     budgetManagerEditable: false,
 });
 
-const handleUpdateBudgetAlerts = async (value: boolean) => {
-    state.updateBudgetAlertsModalVisible = value;
-    await budgetPageStore.getBudgetData(budgetPageState.budgetData?.budget_id ?? '');
-};
+const { mutate: _updateBudget, isPending: isUpdateBudgetPending } = useBudgetUpdateMutation({
+    onSuccess: (_d, _v, __mutationType) => {
+        showSuccessMessage(i18n.t('BILLING.COST_MANAGEMENT.BUDGET.DETAIL.BASE_INFORMATION.UPDATE_SUCCESS', {
+            data: __mutationType.replace(/([A-Z])/g, ' $1').replace(/^./, (s) => s.toUpperCase()),
+        }), '');
+    },
+    onError: (error: any) => {
+        showErrorMessage(error.code, error.message);
+    },
+});
+
+const { mutate: setNotification, isPending: isSetNotificationPending } = useBudgetSetNotificationMutation({
+    onSuccess: (_data, _variables, __mutationType) => {
+        showSuccessMessage(i18n.t('BILLING.COST_MANAGEMENT.BUDGET.DETAIL.BASE_INFORMATION.UPDATE_SUCCESS', {
+            data: __mutationType.replace(/([A-Z])/g, ' $1').replace(/^./, (s) => s.toUpperCase()),
+        }), '');
+    },
+    onError: (error: any) => {
+        showErrorMessage(error.code, error.message);
+    },
+});
 
 const updateBudget = async (updateParams: any, type: string) => {
-    if (storeState.budgetData?.budget_id) {
-        await budgetPageStore.updateBudgetData({
-            budgetId: storeState.budgetData?.budget_id,
-            updateParams,
-        }, type);
+    if (budgetData.value?.budget_id) {
+        _updateBudget({
+            ...updateParams,
+            budget_id: budgetData.value?.budget_id,
+            __mutationType: type,
+        });
```

Comment:
Suggestion: I think use of `__mutationType` in `use-budget-update-mutation` can be quite dangerous.

I think this can not only send wrong data to server internally, but also cause difficulties in debugging or maintenance for other developers who don't know purpose of `__mutationType`.

So, how about using other solution?